crates/turborepo-lib/src/execution_state.rs

```diff
@@ -49,8 +49,22 @@ impl<'a> TryFrom<&'a CommandBase> for ExecutionState<'a> {
 
         let config = base.config()?;
 
+        // TODO(voz): Don't unwrap here. Should be fine since this can rarely ever be
+        // None.
+        let auth_file_path = base.global_auth_path.clone().unwrap();
+        let config_file_path = base.global_config_path.clone().unwrap();
```

Comment:
I'm pretty sure you want 
```suggestion
        let auth_file_path = base.global_auth_path().unwrap();
        let config_file_path = base.global_config_path().unwrap();
```
as the fields themselves will always be `None` except for in tests.

We also can't be unwrapping like this and should instead go through normal error handling if there's a config error.

Reply:
I was doing the `clone` due to move issues, but I can just make those methods public instead.

I'll handle unwraps